app/app.js

```diff
@@ -39,7 +39,10 @@ app.use((err, req, res, next) => {
 });
 
 app.use((err, req, res, next) => {
-  if (err.status && err.msg) {
+  if (err.status === 400 && err.msg === "Bad request") {
+    return res.status(400).send({ msg: "Bad request" });
+  }
+  if (err.status === 404 && err.msg === "not found") {
     return res.status(404).send({ msg: "not found" });
   } else next(err);
```

Comment:
Since all you're doing is sending back the status and the msg from the error, how can you make this dynamic so that whatever status and msg comes in, they get sent back without needing to be hard-coded?